python/tk_multi_publish2/dialog.py

```diff
@@ -143,6 +142,9 @@ def __init__(self, parent=None):
         # start up our plugin manager
         self._plugin_manager = None
 
+        self._summary_comment = ""
+        self._summary_comment_multiple_values = False
```

Comment:
This Boolean indicates that at least one child has a description that is different than the summary.